elpis/engines/hft/objects/transcription.py

```diff
@@ -193,15 +122,17 @@ def _save_utterances(self, utterances) -> None:
         Saves Elan output using the pympi library
         """
         result = pympi.Elan.Eaf(author="elpis")
-
         result.add_linked_file(self.audio_filename)
         result.add_tier("default")
-
         to_millis = lambda seconds: int(seconds * 1000)
-        for word, start, end in zip(*utterances):
-            start, end = to_millis(start), to_millis(end) + 1
+        for utterance in utterances:
+            print(utterance)
```

Comment:
I suspect this line can be removed, but if not:
```suggestion
            logger.info(utterance)
```

Reply:
removed